### PR TITLE
LRQA-70003

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRQA-70003

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test

**Comment by natocesarrego** [comment]
ci:test:stop

**Comment by natocesarrego** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 1]
```diff
@@ -1080,6 +1080,152 @@ definition {
...
+
+		FormFields.viewNumericInputMask(inputMaskValue = "0,00");
+
+		FormFields.editText(
+			fieldName = "Numeric",
+			fieldValue = "1000");
+
+		FormPortlet.submitSuccessfully();
+
+		Refresh();
```
[the remark this answers is not in the document]
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase]: good idea! fixed!! :) @natocesarrego

**Comment by rafaellajordao** [comment]
ci:test:sf

**Pull request closed without merging**